tests.py

```diff
@@ -66,159 +66,172 @@ def test_get_access_token_timeout(self, request_post):
         )
 
     @patch.object(Translator, 'get_access_token', return_value='super-token')
-    @patch('requests.get', return_value=MagicMock(
-        content='<string xmlns="http://schemas.microsoft.com/2003/10/Serialization/">I am tired</string>'
-    ))
-    def test_translate(self, request_get, get_access_token):
+    @patch('requests.Response.raise_for_status', return_value='200')
+    @patch('requests.Response.json', return_value=
+        [ { "translations": [ {"text": "I am tired" } ] } ]
+        )
+    @patch('requests.post')
+    def test_translate(self, request_post, response_json, raise_for_status, get_access_token):
+        request_post.return_value = MagicMock(raise_for_status=raise_for_status, json=response_json)
+
         self.assertEqual(self.translator.translate(text='Je suis fatigué'), 'I am tired')
-        request_get.assert_called_with(
+        request_post.assert_called_with(
             self.translator.TRANSLATE_API,
             headers={
                 'Authorization': 'Bearer super-token',
-                'Accept': 'application/xml',
+                'Accept': 'application/json',
+                'Content-type': 'application/json',
             },
             timeout=Translator.HTTP_TIMEOUT,
             params={
-                'text': 'Je suis fatigué',
                 'to': self.translator.DEFAULT_LANGUAGE,
-            }
+            },
+            json=[{'text': u'Je suis fatigué'}]
         )
-        request_get.return_value.raise_for_status.assert_called_with()
+        request_post.return_value.raise_for_status.assert_called_with()
```

Comment:
ha. Well you don't need to test the call here as you are testing later the raise side_effect of this.

Reply:
@jolancornevin  ok, so just double checking, this call test is completely useless in general(not just for this one call in this method) since we're already testing for something similar.